mautic/mautic
# Connectwise activities

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | y
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/4851
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR writes activities to contacts in connect wise through a Mautic push to integration campaign action.

#### Steps to test this PR:
1. Apply this PR, your connectwise plugin needs to be authorized.
2. create a new campaign with a push to integration action for Connectwise
3. Enter an activity name and type if you wish to create an activity to contacts in the campaign. leave everything blank if you don't wish to push activities.
4. In Connectwise an new activity should be attached to the contact pushed by the campaign action.
